### DISCUSSION TITLE
tag "form" not found

### DISCUSSION THREAD

**Discussion by Phong111222** [harttle/liquidjs#451] [Working with Shopify]

**Comment by wuyonggang100** [comment]
did you resolved this problem ？i have the same question。

**Comment by harttle** [comment]
I can't find it on https://github.com/Shopify/liquid . As described [here](https://liquidjs.com/tutorials/differences.html), Shopify bussiness related features especially those depends on Shopify-specific data are not supposed to be introduced into this repo.

Someone has tried to build shopify-bussiness related plugins for LiquidJS, see #387 #116 #370
